### PR TITLE
Create DroidCam OBS Plugin

### PR DESCRIPTION
**Pull request by aramg**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. 
- [x] All assets referenced in the manifest are redistributable by any party. 
- [x] I am an upstream contributor to the project.
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains
- [ ] Any additional patches or files have been submitted to the upstream projects concerned.

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by hfiguiere** [comment]
cc @GeorgesStavracas

**Comment by hfiguiere** [comment]
bot, build com.obsproject.Studio.Plugin.DroidCam

**Comment by hfiguiere** [comment]
bot, build com.obsproject.Studio.Plugin.DroidCam

**Comment by aramg** [comment]
Ping @GeorgesStavracas

File: com.obsproject.Studio.Plugin.DroidCam.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="addon">
+  <id>com.obsproject.Studio.Plugin.DroidCam</id>
+  <extends>com.obsproject.Studio</extends>
+  <name>DroidCam OBS Plugin</name>
```
Comment (GeorgesStavracas) [inline com.obsproject.Studio.Plugin.DroidCam.metainfo.xml:5]: I'd simplify this to `DroidCam` or `DroidCam Integration`. This name is shown in the "Extensions" section of the OBS Studio page of app stores, like GNOME Software, so we don't have to repeat the "OBS Plugin" information there.

File: com.obsproject.Studio.Plugin.DroidCam.metainfo.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  <summary>Use your phone as a camera source</summary>
```
Comment (GeorgesStavracas) [inline com.obsproject.Studio.Plugin.DroidCam.metainfo.xml:6]:
```suggestion
  <summary>Use your phone as a camera source using the DroidCam app</summary>
```

**Comment by aramg** [comment]
bot, build com.obsproject.Studio.Plugin.DroidCam

**Review by GeorgesStavracas — APPROVED** [review]

**Comment by barthalion** [comment]
This has been merged, there's something wrong with API lately.

**Pull request closed without merging**